Assets/UI/Popups/greatpeoplepopup.lua

```diff
@@ -40,6 +51,21 @@ local CQUI_MARGIN = 180;
 local CQUI_ModalFrameBaseSize = Controls.ModalFrame:GetSizeY();
 local CQUI_PopupContainerBaseSize = Controls.PopupContainer:GetSizeY();
 
+-- These values are calculated as part of the "AddRecruit" function, which is a function separated out from "ViewCurrent" by Firaxis.
```

Comment:
these variables were previously declared inline in the ViewCurrent function, but had to be moved here to accommodate the "AddRecruit" function that generates one of the panel instances for the Great People popup.